src/utils/googleAnalytics.ts

```diff
@@ -9,10 +9,11 @@ export const DEVELOPERS_CATEGORY = 'DEVELOPERS'
 export const FOOTER_CATEGORY = 'FOOTER_CATEGORY'
 
 const getGoogleAnalyticsTrackingID = () => {
-  return typeof window !== 'undefined' &&
+  return (typeof window !== 'undefined' &&
+    window.location.hostname.split('.')[0] === 'safe') ||
     window.location.hostname.split('.')[0] === 'gnosis-safe'
```

Comment:
Why do you need to split the first part instead of just checking the full domain name?

Reply:
Good question. I don't see any reason not to compare the hostname directly. Will change it accordingly